local-cli/generator-ios/templates/app/AppDelegate.m

```diff
@@ -47,6 +47,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                       moduleName:@"<%= name %>"
                                                initialProperties:nil
                                                    launchOptions:launchOptions];
+  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
```

Comment:
Drive-by comment: this is probably more tersely (and idiomatically) expressed as `[UIColor whiteColor]`.